rpcs3/Emu/Cell/lv2/sys_spu.cpp

```diff
@@ -141,6 +141,35 @@ void sys_spu_image::deploy(u32 loc, sys_spu_segment* segs, u32 nsegs)
 	LOG_NOTICE(LOADER, "Loaded SPU image: %s (<- %u)%s", hash, applied, dump);
 }
 
+lv2_spu_group::~lv2_spu_group()
```

Comment:
It's not a destructor's job, it should be in sys_spu_thread_group_destroy

Reply:
The reason it's hrre is because if sys_spu_thread_group_destroy is called, group->threads may be dangling pointers on concurrent access. (notice group->threads are no longer shared_ptr array)

Reply:
It's just wrong. Keep shared_ptr in thread group.

Reply:
It's just wrong. Keep shared_ptr in thread group.

Reply:
Fixed.